### PR TITLE
Contact Us section

### PR DESCRIPTION
**Pull request by reemsh2000**
Relates #7
contact us section with style

### CODE REVIEW

**Review by emanjaber — DISMISSED** [review]
Good job Reem! 
Keep on ^^

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
-# PalArt-Gallery
+#PalArt-Gallery
```
Comment (sara219) [inline README.md]: Always upload each file with the related section and branch. 
like there, you upload the readme.md with a branch and pull-request called contact-us

delete any unrelated files to contact-us

File: index.html [hunk 2]
```diff
@@ -6,12 +6,55 @@
...
     <link rel="stylesheet" href="./css/style.css"/>
+   <link rel="stylesheet" href="./css/font-awesome.min.css">
+   <link rel="stylesheet" href="./css/media.css"/>
     <title>PalArt Gallery</title>
 </head>
 <body>
-
+    <!-- ****************************************************************************** -->
+ <!-- Contact Us section -->
+ <div class="contact-us">
```
Comment (sara219) [inline index.html]: use semantic element 
there's a lot of divs, try to replace them with the semantic element

File: css/style.css [hunk 3]
```diff
@@ -11,4 +11,92 @@ font-family: 'Raleway', sans-serif;
...
+    padding:3%;
+    
+    
+}
+.contact-us-header{
+    font-weight:bold; 
+    color:#487e95 ;
+    margin: center;
+   padding: 3%;
+   font-size: 30px;
```
Comment (sara219) [inline css/style.css]: pay attention to the code formatting, use prettier to format your code 

Peek 2021-03-02 19-15

File: css/style.css [hunk 4]
```diff
@@ -11,4 +11,92 @@ font-family: 'Raleway', sans-serif;
...
+}
+.contact-top{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    /* flex-direction: row; */
```
Comment (sara219) [inline css/style.css]: delete useless and unnecessary line

File: css/style.css [hunk 5]
```diff
@@ -11,4 +11,92 @@ font-family: 'Raleway', sans-serif;
 
 *{
     font-family: 'Raleway', sans-serif;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+}
+
```
Comment (sara219) [inline css/style.css:9]: add a useful comment to  tell us what is that code and the use of it ( to which section also )

**Review by sara219 — CHANGES_REQUESTED** [review]

**Review by sara219 — CHANGES_REQUESTED** [review]
make the width equal, also the colour contrast is a little bit uncomfortable 

imageedit_4_5993238474

**Review by sara219 — APPROVED** [review]

**Review by suhail-radwan — APPROVED** [review]

**Review by emanjaber — APPROVED** [review]

**Review by Heba-elLouh — APPROVED** [review]

**Pull request merged**